src/Dash.jl

```diff
@@ -1,8 +1,9 @@
 module Dash
 using DashBase
-import HTTP, JSON2, CodecZlib, MD5
+import HTTP, JSON2, JSON3, CodecZlib, MD5
```

Comment:
Still need JSON2? (here and in Project.toml)

Reply:
@alexcjohnson fixed at faaf0ae